### PR TITLE
[Tezos Basics] History of evolutions

### PR DESCRIPTION
**Pull request by Max-FER**

### CODE REVIEW

File: docs/tezos-basics/history_of_evolutions.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Notable changes included a new consensus algorithm variant (Emmy+), in addition to new Michelson features to aid smart contract developers, an account rehaul that enabled clearer
+distinction between tz and kt accounts, as well as refinements to the quorum formula and addition of a 5% proposed quorum threshold.
+
+Babylon was autonomously [activated](https://twitter.com/adrian_brink/status/1185137422432161792?s=20) onto the protocol in October 2019.
+
+For a full list of changes be sure to read the corresponding blog posts from [Nomadic Labs](https://blog.nomadic-labs.com/babylon-proposal-injected.html, and [Cryptium Labs](https://medium.com/metastatedev/on-babylon2-0-1-58058d9d2106) (Metastate). 
+
+## [Carthage](https://www.tezosagora.org/proposal/6) (PtCarthav)
+
+The first implementation of the proposal was rejected during the proposal period in november 2019. Since the Babylon change, it takes a minimum of 5% to move to the exploration period and Carthage obtains 3.5%.
```
Comment (bepi-octo) [inline docs/tezos-basics/history_of_evolutions.md]: Carthage obtained 3.5%

File: docs/tezos-basics/history_of_evolutions.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Notable changes included increasing the gas limit per block and per operation by 30%, improving the accuracy of the formula used to calculate baking and endorsing rewards, as well as several minor improvements to Michelson. The main difference with carthage 1 is its new, more secure formula to calculate rewards.
+
+Carthage was autonomously [activated](https://twitter.com/tezos/status/1235590757416751105?s=20) onto the protocol in March 2020.
+
+For a full list of changes be sure to read the corresponding [changelog](https://tezos.gitlab.io/protocols/006_carthage.html#changelog) and blog posts from [Nomadic Labs](https://blog.nomadic-labs.com/carthage-changelog-and-testnet.html) and [Cryptium Labs](https://medium.com/metastatedev/updating-the-potential-carthage-proposal-and-resetting-the-carthagenet-test-network-f413a792571f) (Metastate). 
+
+## [Delphi](https://www.tezosagora.org/proposal/8) (PsDELPH1K)
+The Delphi proposal was injected in September 2020 with contributions by development teams Nomadic Labs, Metastate, and Gabriel Alfour.
```
Comment (bepi-octo) [inline docs/tezos-basics/history_of_evolutions.md]: Between Carthage 2.0 and Delphi, about 8 months have gone by. Does this means that there has not been any other proposal for those 8 months, or some have been proposed, but have been rejected ?

Comment (Max-FER) [reply docs/tezos-basics/history_of_evolutions.md]: no proposals have been sent

Comment (frankhillard) [reply docs/tezos-basics/history_of_evolutions.md]: Shouldn't we change the title to History of Tezos amendments ?

**Pull request merged**